test/unit/model/update.test.js

```diff
@@ -53,4 +53,56 @@ describe(Support.getTestDialectTeaser('Model'), () => {
       await expect(this.User.update(this.updates, { where: new Where() })).to.be.rejected;
     });
   });
+
+  describe('Update with multiple models to the same table', () => {
+    before(function () {
+      this.Model1 = current.define('Model1', {
+        value: DataTypes.INTEGER,
+        name: DataTypes.STRING,
+        isModel2: DataTypes.BOOLEAN,
+        model1ExclusiveData: DataTypes.STRING,
+      }, {
+        tableName: 'model_table',
+      });
+
+      this.Model2 = current.define('Model2', {
+        value: DataTypes.INTEGER,
+        name: DataTypes.STRING,
+      }, {
+        tableName: 'model_table',
+        globalScope: {
```

Comment:
just one note though: we don't have a globalScope option. Did you mean defaultScope? Either way it's not stricly necessary for this test :)

Reply:
oops, my mistake, it's there to make an example of an use case.
fixed in https://github.com/sequelize/sequelize/pull/15231/commits/af63e3d6a5dbb47d31d1ebabd994fec75aea211d